doc/benchmark.dtd

```diff
@@ -15,6 +15,7 @@
 
 <!ELEMENT option (#PCDATA)>
 <!ELEMENT propertyfile (#PCDATA)>
+<!ELEMENT multiproperty (#PCDATA)>
```

Comment:
Fixed in 7604dcc.
In order to use multi-property verification attribute `kind` of tag `propertyfile` should be set to `multiproperty`. Otherwise all properties will be considered as a single composite property.